.taskcluster.yml

```diff
@@ -0,0 +1,36 @@
+version: 0
+tasks:
+  - provisionerId: '{{ taskcluster.docker.provisionerId }}'
+    workerType: '{{ taskcluster.docker.workerType }}'
+    extra:
+      github:
+        events:
+          - pull_request.closed
+          - pull_request.opened
+          - pull_request.reopened
+          - pull_request.synchronize
+          - pull_request.assigned
+          - pull_request.unassigned
+          - pull_request.labeled
+          - pull_request.unlabeled
+          - pull_request.edited
+          - push
+          - release
+          
+    payload:
+      maxRunTime: 3600
+      image: node
+      command:
+        - /bin/bash
+        - '--login'
+        - '-c'
+        - >-
+          git clone {{event.head.repo.url}} repo && cd repo && git config
+          advice.detachedHead false && git checkout {{event.head.sha}} && npm
+          install . && npm test
```

Comment:
We use `yarn` these days -- can you change the npm commands to be `yarn && yarn test` instead?

Reply:
Hi Dustin,
It is  giving failure when I replace npm with yarn.I am changing npm commands  npm
+          install . && npm test with yarn && yarn test.Please suggest what should be done to remove this error.Thank you :)